.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,53 @@
+# Do not run this workflow on pull request since this workflow has permission to modify contents.
+on:
+  push:
+    branches:
+      - master
```

Comment:
Should this be `main` rather than `master`?

Reply:
yep. copy/paste error from example